File: AkitaModelJava/akitamodel/src/main/java/skyware/lis/akitamodel/CashPayment.java
package skyware.lis.akitamodel;

/**
 * Represents a cash payment.
 */
public class CashPayment extends PaymentBase {

    /**
     * Returns the payment method as CASH.
     *
     * @return the payment method.
     */
    @Override
    public PaymentMethod getMethod() {
        return PaymentMethod.CASH;
    }
    
}

File: AkitaMotelTS/src/CashPayment.ts
/**
 * Skyware.Lis.AkitaModel
 */

import { PaymentBase } from './PaymentBase';
import { PaymentMethod } from './PaymentMethod';

export class CashPayment extends PaymentBase {
  /** @inheritdoc */
  get method(): PaymentMethod {
    return PaymentMethod.Cash;
  }
}
